bsp4j/src/main/java/ch/epfl/scala/bsp4j/BuildClient.java

```diff
@@ -90,5 +90,19 @@ public interface BuildClient {
     @JsonNotification("build/taskFinish")
     void onBuildTaskFinish(TaskFinishParams params);
 
+    /**
+     * Notification sent from the server to the client when the target being run prints
```

Comment:
Should it also be used during test?

```suggestion
     * Notification sent from the server to the client when the target being run or tested prints
```